recipes/aws-kvs-pic/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+import os
+from conans import ConanFile, tools, CMake
+
+required_conan_version = ">=1.33.0"
+
+class awskvspicConan(ConanFile):
+    name = "aws-kvs-pic"
+    license = "Apache-2.0"
+    homepage = "https://github.com/awslabs/amazon-kinesis-video-streams-pic"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = ("Platform Independent Code for Amazon Kinesis Video Streams")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {'shared': False, "fPIC": True}
+    generators = "cmake"
+    topics = ("conan", "aws", "kvs", "stream")
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def _configure_cmake(self):
+        if not self._cmake:
+            self._cmake = CMake(self)
+            self._cmake.definitions["BUILD_DEPENDENCIES"] = False
```

Comment:
This option is common across multiple aws libraries, E.g. the one that will be consuming this package (see PR description). It disables building the dependencies from sources and uses standard mechanisms like find_package() to consume them instead.
In this particular repo, it does nothing (even though it is present on the CMake file) because this lib has no dependencies.
I think it is a good idea to explicitly set it to `false` anyways to catch the possible future inclusion of deps.